base.cmake

```diff
@@ -254,3 +254,23 @@ MACRO(SETUP_PROJECT_FINALIZE)
 
   LOGGING_FINALIZE()
 ENDMACRO(SETUP_PROJECT_FINALIZE)
+
+#.rst:
+# .. ifmode:: user
+#
+# .. command:: CHECK_DEBIAN
+#
+#   Checks is the current system is Debian based
+#
+# Variables
+# ---------
+#
+# .. variable:: IS_DEBIAN
+#
+#   **REQUIRED**
+#
+#   A Boolean, indicating if the current system is Debian based
+#
+MACRO(CHECK_DEBIAN IS_DEBIAN)
```

Comment:
I think check_debian should not take any argument but should define a global variable.
You can have a look on https://gist.github.com/lbaehren/2413369